game_eggs/steamcmd_servers/ark_survival_ascended/egg-ark--survival-ascended.json

```diff
@@ -0,0 +1,154 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2023-10-28T17:45:10-04:00",
+    "name": "ARK: Survival Ascended",
+    "author": "blood@darkartsgaming.com",
+    "description": "ARK is reimagined from the ground-up into the next-generation of video game technology with Unreal Engine 5! Form a tribe, tame & breed hundreds of unique dinosaurs and primeval creatures, explore, craft, build, and fight your way to the top of the food-chain. Your new world awaits!",
+    "features": [
+        "steam_disk_space"
+    ],
+    "docker_images": {
+        "ghcr.io\/parkervcp\/steamcmd:proton": "ghcr.io\/parkervcp\/steamcmd:proton"
+    },
+    "file_denylist": [],
+    "startup": "proton run .\/ShooterGame\/Binaries\/Win64\/ArkAscendedServer.exe \"{{SERVER_MAP}}?listen?MaxPlayers={{MAX_PLAYERS}}?SessionName={{SESSION_NAME}}?Port={{SERVER_PORT}}?QueryPort={{QUERY_PORT}}?ServerAdminPassword={{ARK_ADMIN_PASSWORD}}$( [  \"$SERVER_PVE\" == \"0\" ] || printf %s '?ServerPVE=True' )$( [ -z \"$SERVER_PASSWORD\" ] || printf %s '?ServerPassword={{SERVER_PASSWORD}}' ){{ARGS_PARAMS}}\"$( [ \"$BATTLE_EYE\" == \"1\" ] || printf %s ' -NoBattlEye' ) {{ARGS_FLAGS}} & ARK_PID=$! ; sleep 15 ; tail -F .\/ShooterGame\/Saved\/Logs\/ShooterGame.log",
```

Comment:
We can actually pull this off without `sleep` using only `tail` by passing `-c0` to not show previous bytes.

```
tail -c0 -F ./ShooterGame/Saved/Logs/ShooterGame.log --pid=$ARK_PID
```

We can also pass the `$ARK_PID` to `--pid=` to track if the server crashes/exits as it will kill `tail` accordingly.

🎉

Reply:
Interesting, I figured something like this was possible.
I'll test and verify it works and will do a commit for this.